### PR TITLE
feat: add how to create address-validation service plugin doc

### PR DESCRIPTION
**Pull request by brent-hoover**
Signed-off-by: Brent Hoover <brent@thebuddhalodge.com>

Also fixed two broken links.

### CODE REVIEW

**Review by delagroove — APPROVED** [review]

File: how-tos/create-an-address-validation-service.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# How To: Add an Address Validation Service
+
+## Prerequisite Reading
+- [Understanding Plugins](../guides/developers-guide/core/build-api-plugin.md)
+- [Address Validation Operator Guide](../guides/shop-managers-guide/configuring-address-validation.md) FIXME:
+
+## Background
+
+Many types of services such as tax or shipping require that the address be validated so that they know the prices/rates they are quoting
```
Comment (MohanNarayana) [inline how-tos/create-an-address-validation-service.md]: Remove line breaks.

**Review by MohanNarayana — CHANGES_REQUESTED** [review]

**Comment by brent-hoover** [comment]
@MohanNarayana Changes from C/R made

File: guides/create-an-address-validation-service.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+## At A Glance
+Having accurate address information is a critical part of marking sure your customers' product arrives on time and is delivered to the
```
Comment (delagroove) [inline guides/create-an-address-validation-service.md:2]: add spacing

File: guides/create-an-address-validation-service.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+correct customer. In addition, some services such a shipping and tax providers may require you to have a validated address in a specific format. Should
+you need this service, this guide will help create a plugin that interfaces with an address validation service.
+
+In general, to add an address validation service you must do the following:
+- Create a plugin or modify an existing one
+- Create and register an address validation function
+
+## What you need
+- [Understanding Plugins](/developer/open-commerce/guides/build-api-plugin/)
```
Comment (delagroove) [inline guides/create-an-address-validation-service.md:11]: add spacing

File: guides/create-an-address-validation-service.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+  return {
+    suggestedAddresses: validationResults.validatedAddresses,
+    validationErrors: validationResults.messages
+  };
+}
+```
+
+## Format validation results
+More than likely your validation solution returns similar data but in a different schema. In this case you'll need to create a transform function to normalize the data structure for the query. See the `Address` type in the GraphQL schema.
```
Comment (delagroove) [inline guides/create-an-address-validation-service.md:90]: add spacing

Comment (delagroove) [reply guides/create-an-address-validation-service.md:90]: in h2

File: guides/quick-start.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Behind the scenes, the `make` process clones all of the relevant Open Commerce s
 
 When `make` completes, three services will be running on `localhost`:
 
-- Open Commerce API (port 3000), including the [core plugins](/developer/open-commerce/docs/fundamentals/#plugins). This service also contains the GraphQL playground at `localhost:3000/graphql`.
+- Open Commerce API (port 3000), including the [core plugins](/developer/open-commerce/docs/plugins-environment-variables/#plugins). This service also contains the GraphQL playground at `localhost:3000/graphql`.
```
Comment (delagroove) [inline guides/quick-start.md:35]: this change is already in

**Review by delagroove — CHANGES_REQUESTED** [review]

**Comment by brent-hoover** [comment]
Closing to be reopened in a later phase

**Pull request closed without merging**